docs/dg/dev/backend-development/console-commands/console-commands.md

```diff
@@ -223,31 +223,33 @@ To use the npm commands, download and install [Node.js](https://docs.npmjs.com/d
+| ENVIRONMENT          | COMMAND DESCRIPTION                                                                                                                                          | LOCAL COMMAND               | DOCKER CLI COMMAND                                   |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------|------------------------------------------------------|
+| Project/Core         |                                                                                                                                                              |                             |                                                      |
+|                      | Removes all content from the cache directories.                                                                                                              |                             | console cache:empty-all                              |
+|                      | Executes importers (full-import).                                                                                                                            |                             | console data:import                                  |
+|                      | Imports all text translations for both core and project level.                                                                                               |                             | console data:import:glossary                         |
+|                      | Generates transfer objects from transfer XML definition files.                                                                                               |                             | console transfer:generate                            |
+|                      | Generates a cache file for twig templates.                                                                                                                   |                             | console twig:cache:warmer                            |
+| Yves                 | Installs dependencies from the project level.                                                                                                                | npm install                 | console frontend:project:install-dependencies        |
+|                      | Builds both core and project scripts, styles, fonts, and images to the `public/Yves/assets` folder.                                                          | npm run yves                | console frontend:yves:build                          |
+|                      | Runs a build command every time either core or project scripts or style files change during the development.                                                 | npm run yves:watch          |                                                      |
```

Comment:
@pushokwhite What about this one? Also line 250

Reply:
`either core or project`, spells correct, what concerns you?